t/rose-date/00-basic.t

```diff
@@ -346,8 +322,14 @@ file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__OUT__'
 __OUT__
 #-------------------------------------------------------------------------------
 # Test rose date --as-total=FORMAT for use case 2 with an offset
-TEST_KEY=$TEST_KEY_BASE-as-total-between-dates-with-offset
-run_pass "$TEST_KEY" rose date 2000-01-01T00:00:00 --offset=PT1H 2000-01-01T01:00:00 --as-total=s
+python3 -c "import argparse; argparse.ArgumentParser.parse_intermixed_args" &> intermixed_args 
+grep "'ArgumentParser' has no attribute 'parse_intermixed_args'" intermixed_args &> intermixed_args
+if [ $(cat intermixed_args | wc -l) != 0 ]; then
```

Comment:
Python will exit 1 on error which makes this a little easier.

```suggestion
if python3 -c "import argparse; argparse.ArgumentParser.parse_intermixed_args"
```

Reply:
Nice. You learn something every day.